libphobos/libdruntime/core/atomic.d

```diff
@@ -1286,7 +1276,7 @@ else version( GNU )
 {
     import gcc.builtins;
 
-    HeadUnshared!(T) atomicOp(string op, T, V1)( ref shared T val, V1 mod ) nothrow
+    HeadUnshared!(T) atomicOp(string op, T, V1)( ref shared T val, V1 mod ) pure nothrow @nogc @trusted
```

Comment:
Note: we use `@trusted` here, the documentation (and dmd inline asm paths) use `@safe`.